sdk/tables/azure-data-tables/samples/async_samples/sample_encode_dataclass_model_async.py

```diff
@@ -42,50 +51,25 @@ class Car:
     product_id: Optional[UUID] = None
     inventory_count: Optional[int] = None
     barcode: Optional[bytes] = None
-    color: Optional[str] = None
-    maker: Optional[str] = None
+    color: Optional[Color] = None
+    make: Optional[str] = None
     model: Optional[str] = None
     production_date: Optional[datetime] = None
-    big_number: Optional[int] = None
     is_second_hand: Optional[bool] = None
 
 
-class Color(str, Enum):
-    WHITE = "white"
-    GRAY = "gray"
-    BLACK = "black"
-
-
-def encode_int(value):
-    return EdmType.STRING, str(value)
-
-
-def encode_uuid(value):
-    return None, str(value)
-
-
-encoder_map = {
-    int: encode_int,
-    UUID: encode_uuid,
+# Here we will define how certain types should be encoded,
+# in this case, we want to define all Python integers to be treated
+# as int64, and we also want to provide custom encoding for the enum type 'Color'.
+encoder_map: Dict[Type, Union[EdmType, Callable[[Any], Tuple[EdmType, str]]]] = {
```

Comment:
Do we really need to provide the type annotation here? It obscures the meat of what we want to show, no?